guides/common/modules/con_configuring-compliance-policy-deployment-methods.adoc

```diff
@@ -0,0 +1,7 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="configuring-compliance-policy-deployment-methods_{context}"]
```

Comment:
```suggestion
[id="configuring-compliance-policy-deployment-methods"]
```

Might also require some adjustments to xrefs.

Reply:
https://github.com/theforeman/foreman-documentation/issues/4214